testing/set_library_versions.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""
+This file is used by unit testing to specify library versions.
+Commands here should maintain alignment with the environment generated in 
+testing/Dockerfile.
```

Comment:
Add documentation for argument.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/223/commits/34af1684c40bb7e1eb982103ef235afa5ce636d0 .